apis/cloud.redhat.com/v1alpha1/namespacereservation_types.go

```diff
@@ -26,13 +26,16 @@ type NamespaceReservationSpec struct {
 	Duration *string `json:"duration,omitempty"`
 	// Requester is the entity (bot or human) requesting the namespace
 	Requester string `json:"requester"`
+	// PoolType is the pool resource specified to reserve a namespace from
+	Pool string `json:"pool,omitempty" default:"default"`
```

Comment:
This might be a good use case for using an enum of "defualt" or "minimal" 
[Clowder has some examples of that here.](https://github.com/RedHatInsights/clowder/blob/master/apis/cloud.redhat.com/v1alpha1/clowdapp_types.go#L61) 

`default` is new to me though. Sweet!

Reply:
I'll take a look. I didn't see that option before. Thank you!

Reply:
Oh cool, so kubebuilder has a default variant as well:
`// +kubebuilder:default:=default`

If a user doesn't specify a pool then the `default` pool should be set automatically. I like this approach!